layouts/default.vue

```diff
@@ -38,56 +22,26 @@
     </v-navigation-drawer>
 
     <v-main class="app-main">
+      <div v-if="smallScreen" class="route-title-small-screen">{{ currentPageTitle }}</div>
       <slot />
     </v-main>
   </v-app>
 </template>
 
 <style>
-@import "@/assets/styles/main.css";
+  @import "@/assets/styles/main.css";
```

Comment:
this is ultimately a code style choice but i don't think css or js should be indented in their respective blocks in order to save a layer of indentation.  for <template> it's fine because it looks more natural.  end of the day it doesn't matter, just a preference of mine to not indent css or js in components